aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -364,6 +364,7 @@ inline void bgemm_internal_cublaslt(CUDABLAS_BGEMM_ARGTYPES(Dtype)) {
   CuBlasLtMatmulDescriptor computeDesc(computeType, scaleType);
   computeDesc.setAttribute(CUBLASLT_MATMUL_DESC_TRANSA, opa);
   computeDesc.setAttribute(CUBLASLT_MATMUL_DESC_TRANSB, opb);
+  computeDesc.setAttribute<int32_t>(CUBLASLT_MATMUL_DESC_SM_COUNT_TARGET, at::cuda::getCurrentDeviceProperties()->multiProcessorCount - at::globalContext().SMCarveout());
```

Comment:
Maybe only call the API if carveout is specified ? (seems unlikely in the average user case?)